src/modules/audio/audio.c

```diff
@@ -208,9 +205,12 @@ bool lovrAudioInitDevice(AudioType type) {
   ma_device_type deviceType = (type == AUDIO_PLAYBACK) ? ma_device_type_playback : ma_device_type_capture;
 
   ma_device_config config = ma_device_config_init(deviceType);
-  config.sampleRate = LOVR_AUDIO_SAMPLE_RATE;
-  config.playback.format = formats[OUTPUT_FORMAT];
-  config.capture.format = formats[OUTPUT_FORMAT];
+  config.sampleRate = state.config[type].sampleRate;
+  lovrAssert(state.config[AUDIO_PLAYBACK].format == OUTPUT_FORMAT, "Only f32 playback format currently supported");
+  config.playback.format = miniAudioFormatFromLovr[state.config[AUDIO_PLAYBACK].format];
+  config.capture.format = miniAudioFormatFromLovr[state.config[AUDIO_CAPTURE].format];
+  config.playback.pDeviceID = state.config[AUDIO_PLAYBACK].device;
+  config.capture.pDeviceID = state.config[AUDIO_CAPTURE].device;
   config.playback.channels = OUTPUT_CHANNELS;
   config.capture.channels = CAPTURE_CHANNELS;
```

Comment:
Maybe there could be an if here to only set the fields corresponding to the appropriate `AudioType`.  I was briefly surprised about why this was setting both.

Reply:
I followed the existing structure, but yeah, it's clearer to split it up. I've done so now.